charts/jaeger-v2/readme.md

```diff
@@ -0,0 +1,167 @@
+- **readme.md**: Provides chart-specific documentation.
+
+## Helm Installation
+
+You can install the Jaeger Helm chart with the following commands:
+
+### Basic Installation
+
+To install the Jaeger chart using the default values from the included `values.yaml`:
+
+```bash
+helm install <chart_name> ./
+```
+
+- `<chart_name>` is the name you want to give your release.
+
+### Installation with Custom Values
+
+If you want to use a custom configuration, you can pass a custom `values.yaml` file. This allows you to override the default values specified in the chart’s `values.yaml`.
+
+Use the following command to install the chart with a custom values file:
+
+```bash
+helm install <chart_name> ./ -f <custom_values.yaml>
+```
+
+- Replace `<chart_name>` with the desired name for your Helm release.
+- Replace `<custom_values.yaml>` with the path to your custom configuration file.
+
+### Example:
+
+```bash
+helm install jaeger-v2 ./ -f my-config-values.yaml
+```
+
+This command installs the Jaeger chart and uses `my-config-values.yaml` for overriding any configuration specified in the default `values.yaml`.
+
+## Custom Values
+
+When deploying Jaeger using a custom values file, any configuration set in the file will take precedence over the default settings in `values.yaml`. For example, you can modify parameters for `receivers`, `processors`, `exporters`, and more within your custom values file.
+
+### Example Custom `values.yaml`
+
+```yaml
+
+    service:
+      extensions: [jaeger_storage, jaeger_query, remote_sampling, healthcheckv2]
+      pipelines:
+        traces:
+          receivers: [otlp, jaeger, zipkin]
+          processors: [batch, adaptive_sampling]
+          exporters: [jaeger_storage_exporter]
+
+    extensions:
+      healthcheckv2:
+        use_v2: true
+        http: {}
+
+
+      jaeger_query:
+        storage:
+          traces: some_store
+          traces_archive: another_store
+        ui:
+          config_file: ./cmd/jaeger/config-ui.json
+
+      jaeger_storage:
+        backends:
+          some_store:
+            memory:
+              max_traces: 90000
+          another_store:
+            memory:
+              max_traces: 90000
+
+      remote_sampling:
+        adaptive:
+          sampling_store: some_store
+          initial_sampling_probability: 0.2
+        http: {}
+        grpc: {}
+
+    receivers:
+      otlp:
+        protocols:
+          grpc: {}
+          http: {}
+
+      jaeger:
+        protocols:
+          grpc: {}
+          thrift_binary: {}
+          thrift_compact: {}
+          thrift_http: {}
+
+      zipkin: {}
+
+    processors:
+      batch: {}
+      adaptive_sampling: {}
+
+    exporters:
+      jaeger_storage_exporter:
+        trace_storage: some_store
+```
+
+## Dependencies
+
+This Helm chart comes with pre-packaged dependencies for data stores:
+- **Cassandra**: Version `0.15.3`
+- **Elasticsearch**: Version `20.0.4`
```

Comment:
you should probably remove the versions here.  Unless using a template like 'helm-doc' to template and generate these versions, they will get out of date and not aligned.